engine/src/main/java/org/hibernate/validator/internal/engine/messageinterpolation/el/SimpleELContext.java

```diff
@@ -6,74 +6,52 @@
  */
 package org.hibernate.validator.internal.engine.messageinterpolation.el;
 
-import java.lang.reflect.Method;
-
 import javax.el.ArrayELResolver;
 import javax.el.BeanELResolver;
 import javax.el.CompositeELResolver;
-import javax.el.ELContext;
 import javax.el.ELResolver;
 import javax.el.ExpressionFactory;
 import javax.el.ListELResolver;
 import javax.el.MapELResolver;
 import javax.el.ResourceBundleELResolver;
-import javax.el.ValueExpression;
-import javax.el.VariableMapper;
+import javax.el.StandardELContext;
+import javax.el.StaticFieldELResolver;
 
 /**
  * @author Hardy Ferentschik
+ * @author Guillaume Smet
  */
-public class SimpleELContext extends ELContext {
+public class SimpleELContext extends StandardELContext {
```

Comment:
Or, if we want to avoid `ELManager` due to the classloading issues we discussed a while ago, why is it that we need a custom `ELContext`?